specification/app/data-plane/Microsoft.App/preview/2023-08-01-preview/containerapps.json

```diff
@@ -0,0 +1,806 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft.App service",
+    "version": "2023-08-01-preview",
+    "x-typespec-generated": [
+      {
+        "emitter": "@azure-tools/typespec-autorest"
+      }
+    ]
+  },
+  "schemes": [
+    "https"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "AadOauth2Auth": [
+        "https://dynamicsessions.io/.default"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "AadOauth2Auth": {
+      "type": "oauth2",
+      "description": "The Azure Active Directory OAuth2 Flow",
+      "flow": "accessCode",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "scopes": {
+        "https://dynamicsessions.io/.default": ""
+      },
+      "tokenUrl": "https://login.microsoftonline.com/common/oauth2/token"
+    }
+  },
+  "tags": [],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/sessionPools/{sessionPoolName}/code/execute": {
+      "post": {
+        "operationId": "CodeExecution_ExecuteCode",
+        "description": "The most basic operation.",
```

Comment:
<html>
<body>
<!--StartFragment-->
Description
--
code/execute | POST | Execute code in a session.

<!--EndFragment-->
</body>
</html>

Reply:
Description fixed